package.json

```diff
@@ -39,11 +39,13 @@
   },
   "homepage": "https://github.com/ethereum/solc-js#readme",
   "dependencies": {
+    "command-exists": "^1.2.8",
     "fs-extra": "^0.30.0",
     "keccak": "^1.0.2",
     "memorystream": "^0.3.1",
     "require-from-string": "^2.0.0",
     "semver": "^5.5.0",
+    "tmp": "0.0.33",
```

Comment:
Based on this version number this seems to be a non-maintained dependency. We have to be careful what kind of dependencies we take.

Reply:
Do you know any other lib for this? When I searched it seemed the most common

Reply:
Ok it has a fair amount of users.